vnext/Microsoft.ReactNative/DesktopWindowMessage.idl

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.ReactNative {
+  [default_interface][webhosthidden] runtimeclass DesktopWindowMessage {
+    UInt32 uMsg {
+      get;
+    };
+    UInt64 wParam {
+      get;
+    };
+    Int64 lParam {
+      get;
+    };
```

Comment:
Please manually format the code as we do in other IDL files. The Clang format used by Visual Studio does not understand the MIDL 3.0 syntax and makes quite random decisions how to format the code.